deploy_freenas.py

```diff
@@ -20,15 +20,20 @@
 import requests
 import subprocess
 from datetime import datetime
+from urllib3.exceptions import InsecureRequestWarning
+requests.packages.urllib3.disable_warnings(category=InsecureRequestWarning)
 
-PRIVATEKEY_PATH = "/root/.acme.sh/your_fqdn/your_fqdn.key"
-FULLCHAIN_PATH = "/root/.acme.sh/your_fqdn/fullchain.cer"
-USER = "root"
-PASSWORD = "ReallySecurePassword"
 DOMAIN_NAME = "your_fqdn"
+PASSWORD = "ReallySecurePassword"
+
+USER = "root"
+PRIVATEKEY_PATH = "/root/.acme.sh/" + DOMAIN_NAME + "/" + DOMAIN_NAME + ".key"
+FULLCHAIN_PATH = "/root/.acme.sh/" + DOMAIN_NAME + "/fullchain.cer"
 PROTOCOL = 'http://'
+PORT = '80'
 now = datetime.now()
-cert = "letsencrypt-%s-%s-%s" %(now.year, now.strftime('%m'), now.strftime('%d'))
+cert = "letsencrypt-%s-%s-%s-%s" %(now.year, now.strftime('%m'), now.strftime('%d'), ''.join(c for c in now.strftime('%X') if
```

Comment:
this allows us to upload more than once in the same day by adding the time to the cert name.

I also personally chose to use my FQDN instead of "letsencrypt" but didn't see a need to commit that change